dashboard-for-web/articles/web-dashboard-designer-mode/data-analysis/dashboard-parameters/passing-parameter-values.md

```diff
@@ -18,7 +18,7 @@ The Web Dashboard provides the capability to bind a dashboard parameter and the
 
```

Comment:
Line 7: In this topic, it describes how to pass the created dashboard parameter to the dashboard. For instance, you can include a dashboard parameter to a _WHERE_ clause of the SQL query or you can filter a dashboard dynamically according to the required parameter value(s).

Can we update this as follows?

`This topic describes how to pass the created dashboard parameter to the dashboard. For instance, you can include a dashboard parameter in the the SQL query's _WHERE_ clause or you can filter a dashboard dynamically according to the required parameter value(s).`